docker-app/qfieldcloud/core/views/files_views.py

```diff
@@ -167,13 +166,9 @@ def post(self, request, projectid, filename, format=None):
 
         request_file = request.FILES.get("file")
 
-        file_size_bytes = request_file.size
-        quota_left_bytes = project.owner.useraccount.storage_free_bytes
-
-        if file_size_bytes > quota_left_bytes:
-            raise QuotaError(
-                f"Requiring {file_size_bytes} bytes of storage but only {quota_left_bytes} bytes available."
-            )
+        permissions_utils.check_can_upload_file(
+            project, request.auth.client_type, request_file.size
+        )
```

Comment:
Eplanation: Before we always raised QuotaError if file too big. Now we ignore if client is `qfield` or `worker`